core/src/main/java/io/kestra/core/models/tasks/retrys/AbstractRetry.java

```diff
@@ -58,4 +61,9 @@ public static <T> RetryPolicy<T> retryPolicy(AbstractRetry retry) {
         return new RetryPolicy<T>()
             .withMaxAttempts(1);
     }
+
+    public enum Behavior {
+        RETRY_FAILED,
```

Comment:
```suggestion
        RETRY_FAILED_TASKS,
```

I think it seems worth making it more verbose but explicit - WDYT?